helm/prometheus-rules/templates/alerting-rules/cluster-autoscaler.rules.yml

```diff
@@ -16,7 +16,7 @@ spec:
       annotations:
         description: '{{`Cluster-Autoscaler on {{ $labels.cluster_id }} has increased errors.`}}'
         opsrecipe: cluster-autoscaler-scaling/
-      expr: increase(cluster_autoscaler_errors_total[10m]) > 0
+      expr: increase(cluster_autoscaler_errors_total{type="cloudProviderError"}[10m]) > 0
```

Comment:
since we added the two new alerts, I would ditch this one.
as long as pods are scheduled and there are no unneeded nodes we don't care if cluster autoscaler is erroring.

wdyt?

Reply:
Sounds good let me revert this